src/controllers/searchController.js

```diff
@@ -0,0 +1,58 @@
+
+import model from "../database/models";
+import Sequelize from 'sequelize'
+
+const Op = Sequelize.Op;
+
+const returnQueries = (query) =>{
+    const queries = [];
+    for (const key in query) {
+      const value = query[key];
+      
+      if (key === 'requestStatus') queries.push({ [key]: { [Op.iLike]: `%${value}%` } }) ;
```

Comment:
Please try to query or search a request using all possible string fields (i.e. requesterName, hotelName, location, roomType)